content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Pods And Endpoints Termination Flow
+content_type: tutorial
+weight: 60
+---
+
+
+<!-- overview -->
+
+## The Kubernetes model for connecting containers
+
+Now that you have a continuously running, replicated application, that is exposed on a network, let's look at Pods termination flow and ways to implement a graceful connections draining.
```

Comment:
In a new tutorial page like this, we may want to polish the first sentence.

Also, please wrap long lines when appropriate.

Reply:
I wrapped lines. Is there any suggestions on how to make text better?